Stack/Opc.Ua.Core/Stack/Tcp/UaSCBinaryChannel.Symmetric.cs

```diff
@@ -180,21 +191,33 @@ protected void ComputeKeys(ChannelToken token)
                 SecurityPolicyUri == SecurityPolicies.Aes128_Sha256_RsaOaep ||
                 SecurityPolicyUri == SecurityPolicies.Aes256_Sha256_RsaPss)
```

Comment:
should we maybe invert this if statement  (make SHA2 the else case)  so that for future Security Policies we don´t use SHA1 by mistake?

Reply:
good point, I simplified the function for a bit.